match_file.csv

```diff
@@ -1,26 +1,42 @@
 EXT	FG
 C3_BREAST_EXALLC.csv	C3_BREAST_EXALLC
 C3_PROSTATE_EXALLC.csv	C3_PROSTATE_EXALLC
+C3_PROSTATE_EXALLC_withp_alleles_eur.csv	C3_PROSTATE_EXALLC
 C3_SKIN_EXALLC.csv	C3_SKIN_EXALLC
+C3_COLON_EXALLC.csv	C3_COLON_EXALLC
+C3_BRONCHUS_LUNG_EXALLC.csv	C3_BRONCHUS_LUNG_EXALLC
 H7_GLAUCPRIMOPEN.csv	H7_GLAUCPRIMOPEN
 G6_ALZHEIMER.csv	G6_ALZHEIMER
 G6_ALZHEIMER.csv	G6_AD_WIDE
 G6_MS.csv	G6_MS
 G6_PARKINSON.csv	G6_PARKINSON
 pd_no_proxy_2017.csv	G6_PARKINSON
+pd_no_proxy_2017.csv	PDSTRICT
+pd_no_proxy_2017.csv	PD2ND_EXMORE
 H7_AMD.csv	H7_AMD
 I9_AF.csv	I9_AF
 I9_HYPERTENSION.csv	I9_HYPERTENSION
 IBD_CD.csv	K11_IBD_STRICT
 IBD_IBD.csv	K11_IBD_STRICT
 IBD_UC.csv	K11_IBD_STRICT
 IBD_UC.csv	K11_ULCER
+IBD_UC.csv	K11_UC_STRICT0
 IBD_UC.csv	K11_UC_STRICT
 J10_ASTHMA.csv	J10_ASTHMA
 J10_COPD.csv	J10_COPD
 L12_ATOPIC.csv	L12_ATOPIC
 L12_PSORIASIS.csv	L12_PSORIASIS
 M13_ANKYLOSPON.csv	M13_ANKYLOSPON
+M13_ANKYLOSPON.csv	M13_ANKYLOSPON_STRICT
 T1D.csv	T1D
 T2D.csv	T2D
 RHEUMA_SEROPOS.csv	RHEUMA_SEROPOS
+melanoma.csv	C3_SKIN_EXALLC
```

Comment:
For skin cancer there are now 3 files: 
- C3_SKIN_EXALLC.csv
- melanoma.csv
- basal_cell_carcinoma.csv
What is the purpose of having both melanoma.csv and C3_SKIN_EXALLC.csv? 
They are covering the same phenotype and one could be removed.

Reply:
We don't have C3_SKIN_EXALLC.csv
We just compare that scan of ours to melanoma.csv and basal_cell_carcinoma.csv

Reply:
So C3_SKIN_EXALLC.csv has everything combined and then we have subtypes. Just for comparison purposes as our phenotype has mix of both

Reply:
Ok, so we keep all 3 files and C3_SKIN_EXALLC.csv contains melanoma and basal_cell_carcinoma.